scripts/install

```diff
@@ -1,49 +1,59 @@
 #!/bin/bash
 
-source _common.sh
-source /usr/share/yunohost/helpers
-
 #=================================================
-# INSTALL DEPENDENCIES
+# IMPORT GENERIC HELPERS
 #=================================================
-ynh_script_progression "Installing dependencies..."
 
-ynh_nodejs_install
+source _common.sh
+source /usr/share/yunohost/helpers
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 ynh_script_progression "Setting up source files..."
 
+# Download, check integrity, uncompress and patch the source from manifest.toml
 ynh_setup_source --dest_dir="$install_dir"
 
+#=================================================
+# INSTALL DEPENDENCIES
+#=================================================
+ynh_script_progression "Installing dependencies..."
+
+ynh_nodejs_install
+
 #=================================================
 # SYSTEM CONFIGURATION
 #=================================================
 ynh_script_progression "Adding system configurations related to $app..."
 
+# Create a dedicated NGINX config using the conf/nginx.conf template
 ynh_config_add_nginx
 
+# Create a dedicated systemd config
 ynh_config_add_systemd
-
 yunohost service add "$app" --description="Legible badges in SVG" --log="/var/log/$app/$app.log"
 
+# Use logrotate to manage application logfile(s)
+ynh_config_add_logrotate
+
 #=================================================
 # INSTALL SHIELDS
 #=================================================
 ynh_script_progression "Installing $app..."
 
 pushd "$install_dir"
  	ynh_hide_warnings ynh_exec_as_app npm ci
- 	ynh_hide_warnings ynh_exec_as_app npm run build
+ 	ynh_hide_warnings ynh_exec_as_app NODE_OPTIONS="--max-old-space-size=3000" npm run build
 popd
 
 #=================================================
 # START SYSTEMD SERVICE
 #=================================================
 ynh_script_progression "Starting $app's systemd service..."
 
-ynh_systemctl --service="$app" --action="start" --log_path="systemd"
+# Start a systemd service
+ynh_systemctl --service="$app" --action="start" --wait_until="Server is starting up" --log_path="/var/log/$app/$app.log"
```

Comment:
is `--wait_until="Server is starting up"` really needed?

Reply:
it's what is written in the logs